workflows/stairway.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, workdir, stairway_dir, java_exe="java"):
         self.classpath = "{}:{}".format(stairway_path, stairway_path / "swarmops.jar")
         self.java_exe = java_exe
 
-    def ts_to_stairway(self, ts_path, selection, num_bootstraps=1, mask_file=None):
+    def ts_to_stairway(self, ts_path, num_bootstraps=1, mask_file=None):
```

Comment:
I need to update this part by extracting the nonsynonymous part of the SFS. I will open a PR for that once this one is in.